roms_tools/setup/datasets.py

```diff
@@ -118,8 +123,17 @@ def load_data(self) -> xr.Dataset:
         ------
         FileNotFoundError
             If the specified file does not exist.
+        ValueError
+            If wildcards are found in the filename and use_dask=False.
         """
 
+        # Check for wildcards in the filename
+        if not self.use_dask and re.search(r"[\*\?\[\]]", self.filename):
+            raise ValueError(
+                "Wildcards detected in the filename but use_dask=False. "
+                "Wildcards can only be used when use_dask=True."
+            )
```

Comment:
Should probably leave a note for devs here that links to https://github.com/pydata/xarray/issues/9038

Reply:
Done in [d6db2f6](https://github.com/CWorthy-ocean/roms-tools/pull/121/commits/d6db2f6665403778ba466b928dd3e3daa3e7d3f3)